### PR TITLE
ZK-4760: default error message box hard coded width (too small)

### PR DESCRIPTION
**Pull request by b952426**
ZK-4760: default error message box hard coded width (too small)

### CODE REVIEW

File: zul/src/archive/web/zul/less/_zkvariables.less [hunk 1]
```diff
@@ -191,10 +191,15 @@
...
+@messageboxMargin:             4px 0px 16px;
 @messageboxWindowPadding:      0;
+@messageboxWindowWidth:        480px;
 @messageboxHeaderPadding:      @containerPadding;
-@messageboxInnerPadding:       @containerPadding;
+@messageboxInnerPadding:       20px 0px 16px;
 @messageboxButtonsMarginLeft:  8px;
+@messageboxSeparatorHeight:    16px;
+@messageboxIconMarginLeft:     24px;
+@messageboxViewportWidth:      477px;
```
Comment (b952426) [inline zul/src/archive/web/zul/less/_zkvariables.less:202]: remove 477

File: zul/src/archive/web/zul/less/profiles/_compact.less [hunk 2]
```diff
@@ -22,14 +22,19 @@
 @containerPadding: 5px;
 @containerButtonPadding: 0;
 @panelHeaderPadding: 7px 5px 5px;
-@windowHeaderPadding: 3px 0 5px;
+@windowHeaderPadding: 3px 8px 5px;
```
Comment (b952426) [inline zul/src/archive/web/zul/less/profiles/_compact.less:25]: use css

File: zul/src/archive/web/zul/less/profiles/_compact.less [hunk 3]
```diff
@@ -22,14 +22,19 @@
...
 @windowGhostHeaderPadding: 8px 5px 5px;
-@messageboxWindowPadding: @containerPadding;
+@messageboxMargin: 2px 0px 12px;
+@messageboxWindowWidth: 360px;
 @messageboxHeaderPadding: @windowHeaderPadding;
-@messageboxInnerPadding: 16px;
+@messageboxInnerPadding: 16px 0px;
 @messageboxButtonsMarginLeft: 4px;
+@messageboxSeparatorHeight: 12px;
+@messageboxIconMarginLeft:16px;
```
Comment (b952426) [inline zul/src/archive/web/zul/less/profiles/_compact.less:33]: @messageboxIconMarginLeft: 16px;

File: zul/src/archive/web/zul/less/profiles/_compact.less [hunk 4]
```diff
@@ -22,14 +22,19 @@
...
 @pagingInputPadding: 0 8px;
 @pagingOsButtonPadding: 4px 3px;
+@messageboxViewportWidth:357px;
```
Comment (b952426) [inline zul/src/archive/web/zul/less/profiles/_compact.less:36]: remove

**Review by scribetw — CHANGES_REQUESTED** [review]

File: zul/src/archive/web/zul/less/_zkvariables.less [hunk 5]
```diff
@@ -191,10 +191,14 @@
 @windowHeaderPadding:          0 0 16px 0;
 @windowGhostHeaderPadding:     @containerPadding;
 // Messagebox
+@messageboxPadding:        4px 16px 16px;
 @messageboxWindowPadding:      0;
-@messageboxHeaderPadding:      @containerPadding;
-@messageboxInnerPadding:       @containerPadding;
+@messageboxWindowWidth:        480px;
+@messageboxHeaderPadding:      18px 16px;
+@messageboxWindowContentPadding:       20px 0px 16px;
```
Comment (b952426) [inline zul/src/archive/web/zul/less/_zkvariables.less:198]: @messageboxWindowContentPadding:  20px 0px 16px;

File: zul/src/archive/web/zul/less/_zkvariables.less [hunk 6]
```diff
@@ -191,10 +191,14 @@
...
 @messageboxButtonsMarginLeft:  8px;
+@messageboxSeparatorHeight:    16px;
```
Comment (b952426) [inline zul/src/archive/web/zul/less/_zkvariables.less:200]: viewport: margin-bottom:

File: zul/src/archive/web/zul/less/profiles/_compact.less [hunk 7]
```diff
@@ -25,9 +25,13 @@
...
 @windowGhostHeaderPadding: 8px 5px 5px;
 @messageboxWindowPadding: @containerPadding;
-@messageboxHeaderPadding: @windowHeaderPadding;
-@messageboxInnerPadding: 16px;
+@messageboxWindowWidth: 360px;
+@messageboxHeaderPadding: 5px 8px;
+@messageboxWindowContentPadding: 16px 0px;
+@messageboxPadding: 2px 16px 12px;
 @messageboxButtonsMarginLeft: 4px;
+@messageboxSeparatorHeight: 12px;
```
Comment (b952426) [inline zul/src/archive/web/zul/less/profiles/_compact.less:33]: remove

**Review by DevChu — APPROVED** [review]

**Review by plijyfes — APPROVED** [review]

**Review by scribetw — APPROVED** [review]

**Comment by scribetw** [comment]
@zkadmin rebase

**Comment by zkadmin** [comment]
I just pushed up the changes, enjoy!

**Pull request merged**